fortitude/src/check.rs

```diff
@@ -640,6 +660,19 @@ pub fn check(args: CheckArgs, global_options: &GlobalConfigArgs) -> Result<ExitC
         extend_select: args.extend_select.unwrap_or(file_settings.extend_select),
     };
 
+    let mut per_file_ignores = args
+        .per_file_ignores
+        .or(file_settings.per_file_ignores)
+        .unwrap_or_default();
+    per_file_ignores.extend(
```

Comment:
Does this not work?

```suggestion
    let per_file_ignores = args
        .per_file_ignores
        .or(file_settings.per_file_ignores)
        .unwrap_or_default()
        .extend(
```

Reply:
I couldn't get that to work, but there was a similar iterator trick to avoid the mutable intermediate